contracts/Lock.sol

```diff
@@ -90,8 +91,13 @@ contract Lock is AragonApp, IForwarder, IForwarderFee {
     * @return Forwarder token address
     * @return Forwarder lock amount
     */
-    function forwardFee() external view returns (address, uint256) {
-        return (address(token), lockAmount);
+    function forwardFee() external view auth(LOCK_TOKENS_ROLE) returns (address, uint256) {
+        (uint256 _griefAmount, ) = getGriefing(msg.sender);
+
+        //add base amount to grief
+        _griefAmount = _griefAmount.add(lockAmount);
```

Comment:
Create a new uint256 and call this something like `totalLockAmountRequired`. Then you can remove the comment above it.